notebooks/Add GPS Metadata Fields.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.                        return "ESRI_FIX_TYPE_DOMAIN is missing a coded value pair."
</pre><p>I would expect this to raise an Exception instead of returning a string.</p><p><code>raise Exception("ESRI_FIX....")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/field-maps-scripts/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>